export/wiki/items/status.py

```diff
@@ -17,32 +42,33 @@ def convert_status_effect(entry):
     pctOfMax = d['bPercentOfMaxStatusValue']
     pctOfCur = d['bPercentOfCurrentStatusValue']
     if pctOfCur or pctOfMax:
-        result['pctOf'] = 'max' if pctOfMax else 'current'
-        result['pctAbsRange'] = (d['PercentAbsoluteMinValue'], d['PercentAbsoluteMaxValue'])
-
-    if d['bSetValue']:
-        result['setValue'] = True
-    if d['bSetAdditionalValue']:
-        result['setAddValue'] = True
-    if d['bForceUseStatOnDinos']:
-        result['forceUseOnDino'] = True
-    if not d['bDontRequireLessThanMaxToUse']:
-        result['allowWhenFull'] = False
-
-    qlyMult = d['ItemQualityAddValueMultiplier']
-    if qlyMult != 1.0:
-        result['qualityMult'] = d['ItemQualityAddValueMultiplier']
+        result.pctOf = 'max' if pctOfMax else 'current'
+        result.pctAbsRange = (d['PercentAbsoluteMinValue'], d['PercentAbsoluteMaxValue'])
 
     if d['bAddOverTimeSpeedInSeconds']:
-        result['duration'] = d['AddOverTimeSpeed']
+        result.duration = d['AddOverTimeSpeed']
     else:
         if d['AddOverTimeSpeed'] == 0:
             # Assume instant
-            result['duration'] = 0
+            result.duration = 0
         else:
             # Duration = amount / speed
             duration = abs(d['BaseAmountToAdd']) / d['AddOverTimeSpeed']
             duration = float(format(duration, '.7g'))
-            result['duration'] = duration
+            result.duration = duration
 
     return (stat_name, result)
+
+
+def convert_status_effects(item: PrimalItem) -> Dict[str, StatEffectData]:
+    # Behavior verified 26/04/2021.
```

Comment:
Just a tip, no need to change: To avoid confusion in open forums use yyyy/mm/dd format.